src/main/modules/Output.js

```diff
@@ -0,0 +1,19 @@
+export class Response {
```

Comment:
Change class name to Output

Reply:
Changed the name of the class Response to Output